common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4308,6 +4307,15 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         + "," + FileBasedKeyStoresFactory.resolvePropertyName(SSLFactory.Mode.SERVER, FileBasedKeyStoresFactory.SSL_TRUSTSTORE_RELOAD_INTERVAL_TPL_KEY)
         + "," + FileBasedKeyStoresFactory.resolvePropertyName(SSLFactory.Mode.SERVER, FileBasedKeyStoresFactory.SSL_TRUSTSTORE_TYPE_TPL_KEY)
         + "," + FileBasedKeyStoresFactory.resolvePropertyName(SSLFactory.Mode.SERVER, FileBasedKeyStoresFactory.SSL_EXCLUDE_CIPHER_LIST)
+        // these properties should not be used by Hive, that's why there is no enum. However they are in the ssl-server.xml
+        // which is available to the daemons. As such we should avoid to print them when a user requests a property dump or
+        // value for a specific field.
+        + ",hops.jwt-manager.master-token"
+        + ",hops.jwt-manager.master-token-0"
```

Comment:
The one time JWTs are called `renew-token-%i` Only the master is called `master-token`.

Reply:
Thanks for catching